alma/lib/Helpers/ThemeHelper.php

```diff
@@ -54,21 +54,36 @@ public function __construct()
     /**
      * @return string
      */
-    public function getThemeNameWithVersion()
+    public function getThemeName()
     {
-        $themeName = $this->contextFactory->getContext()->shop->theme_name;
+        return $this->contextFactory->getContext()->shop->theme_name;
+    }
+
+    /**
+     * @return string
+     */
+    public function getThemeVersion()
+    {
+        $themeVersion = '';
         $themeConfigPath = self::CONFIG_THEME_FILE;
 
-        // WARNING : NOT COMPATIBLE WITH PS 1.6
-        // TODO : Need to explo a better compatibility with PS 1.6
         if ($this->toolsHelper->psVersionCompare('1.7', '>=')) {
             if (file_exists($themeConfigPath)) {
                 $themeConfig = \Symfony\Component\Yaml\Yaml::parseFile($themeConfigPath);
                 $themeVersion = $themeConfig['version'] ?: 'undefined';
-                $themeName = $themeName . ' ' . $themeVersion;
+            }
+        } else {
+            $themeDirectory = _PS_THEME_DIR_;
+            $configFilePath = $themeDirectory . 'config.xml';
+
+            if (file_exists($configFilePath)) {
+                $xmlContent = simplexml_load_file($configFilePath);
```

Comment:
We need XML extension. is this already mandatory for Prestashop?

Reply:
simplexml_load_file is a native PHP function so we don't need a XML extension I think

Reply:
The extension "ext-simplexml" is include in Prestashop (1.6 and 1.7)